### DISCUSSION TITLE
Can't use PNPM

### DISCUSSION THREAD

**Discussion by sotnikovse** [vuejs/vitepress#1441] [Q&A] [answered]
[Contributing Guide](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md#development-setup) says PNPM is needed, but I can't use PNPM.

Tried to run with yarn and npm, but no luck, `pnpm: command not found`.

How can I use yarn or npm?

**Comment by brc-dd** [comment, marked as answer]
> but I can't use PNPM.

Why? In newer node versions you just need to run `corepack enable` to use pnpm.

**Reply by sotnikovse** [reply]
No have access to pnpm docs, pnpm.io is blocked for Russia and Belarus (https://github.com/pnpm/pnpm/issues/4498#issuecomment-1082440873)

`corepack enable` works for me

Thank You!
